src/client/js/script.js

```diff
@@ -147,7 +147,7 @@ function renderCard(i, noFetched, template, eventtitle, eventimage, eventvenue,
   startdate, startmonth, startyear, starthours, startminutes, eventdescription, eventid, eventurl, stateofevent){
   var newname = template.replace('alt=""', 'alt="' + eventtitle + '"');
   var newurl = newname.replace('src=""', 'src="' + eventimage + '"');
-  var newvenue = newurl.replace('<span></span>', '<span>' + eventvenue + '</span>');
+  var newvenue = newurl.replace('<span></span>', '<span class="text-check">' + eventvenue + '</span>');
```

Comment:
Won't this work if we add this class directly to the the html template? And would love to have this on the title as well. Since, we're already loading an external file for this.